config.json

```diff
@@ -9,5 +9,5 @@
     "T2D_REDIRECT_URI": "http://localhost:3000",
     "T2D_DISCORD_CHANNEL": "",
     "T2D_DISCORD_TOKEN": "",
-    "T2D_DEV_SAVE_TOKEN": false
+    "T2D_DEV_SAVE_TOKEN": true
```

Comment:
We'll need to keep this as false - in a production instance it would be extremely rare to be rebooting quickly enough to have the same token

Reply:
Accident, had to remove my tokens like 3 times cause the gitignore is garbo

Reply:
That gitignore is very weird XP